src/arch/mips/kernel/entry.S

```diff
@@ -44,9 +55,9 @@ NESTED(mips_first_exception_handler, 0, $sp)
 	.set  push                 /* save the current status of flags */
 
 	mfc0  $k1, $CP0_CAUSE
-	andi  $k1, $k1, 0x7c       /* read exception number */
-
-	j     mips_second_exception_handler /* jump to real exception handler */
+/*	andi  $k1, $k1, 0x7c        read exception number */
```

Comment:
We can remove this line since it's not used anymore (migrated to mips_second_exception_handler)

Reply:
done